includes/Core/Authentication/Google_Proxy.php

```diff
@@ -54,6 +54,107 @@ public function __construct( Context $context ) {
 		$this->context = $context;
 	}
 
+	/**
+	 * Returns the application name: a combination of the namespace and version.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The application name.
+	 */
+	public static function get_application_name() {
+		return 'wordpress/google-site-kit/' . GOOGLESITEKIT_VERSION;
+	}
+
+	/**
+	 * Gets the list of features to declare support for when setting up with the proxy.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Array of supported features.
+	 */
+	private function get_supports() {
+		$supports = array(
+			'credentials_retrieval',
+			'short_verification_token',
+			'user_input_flow',
```

Comment:
We need to update this to also include the feature which was just added here:
https://github.com/google/site-kit-wp/blob/baec79d391492896a1ac4d0b41a6c93ac67ccf5c/includes/Core/Authentication/Clients/OAuth_Client.php#L921-L924